src/Domain/FaceNormal.hpp

```diff
@@ -118,13 +118,12 @@ struct UnnormalizedFaceNormalMovingMeshCompute
       tmpl::list<Mesh<VolumeDim - 1>, ElementMap<VolumeDim, Frame::Grid>,
                  CoordinateMaps::Tags::CoordinateMap<VolumeDim, Frame::Grid,
                                                      Frame::Inertial>,
-                 ::Tags::Time, domain::Tags::FunctionsOfTime,
-                 Direction<VolumeDim>>;
+                 ::Tags::Time, Tags::FunctionsOfTime, Direction<VolumeDim>>;
```

Comment:
you can actually also remove the `Tags::` namespace as you did for the other domain tags. I'm not asking you to change that though, it's insignificant and I don't even know if I like it better with or without the namespaces.